src/NServiceBus.Transport.SQS/ReducedPayloadSerializerConverter.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace NServiceBus.Transport.SQS
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json;
+    using System.Text.Json.Serialization;
+
+    /// <summary>
+    /// This serializer is only used to make sure we are not double storing TimeToBeReceived and ReplyAddress both in
+    /// the headers and on the transport message wrapper to save space on the SQS message body.
+    /// The read method is implemented but never used because only the dispatcher uses it to serialize the transport
+    /// message.
```

Comment:
```suggestion
    /// This converter is only used to make sure we are not double storing TimeToBeReceived and ReplyAddress both in
    /// the headers and on the transport message wrapper to save space on the SQS message body.
    /// The read method is implemented but never used because only the dispatcher uses it to serialize the transport
    /// message.
```